docs/index.html

```diff
@@ -36,6 +36,7 @@ <h1><a href="https://threejs.org">three.js</a></h1>
 						<option value="ko">한국어</option>
 						<option value="zh">中文</option>
 						<option value="ja">日本語</option>
+						<option value="pt-br">pt-br</option>
```

Comment:
Shouldn't the text be the translation of "Brazilian Portugues" to Portuguese rather than just `pt-bt` string?

Reply:
I think this is the pattern, like the other languages in the file

Reply:
I think the pattern is using the language name translated to that language.

Reply:
Not necessarily, we have "en" and "ar" in the first options. Also, i think "português brasileiro" would be too long compared to the others options